src/helpers/date.js

```diff
@@ -0,0 +1,39 @@
+export const defaultDate = () => {
```

Comment:
This looks great I was researching for a dependency to do this for us. Great job